src/base/bittorrent/session.cpp

```diff
@@ -3507,6 +3507,7 @@ void Session::handleTorrentResumeDataReady(TorrentHandle *const torrent, const l
     // isn't cheap too.
 
     QByteArray out;
+    out.reserve(100 * 1024);  // most fastresume file sizes are under 100 KB
```

Comment:
I won't approve it yet. Let's wait for other opinions and think about it more carefully.